health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/RenderConditionalField.js

```
@@ -0,0 +1,220 @@
+
+          <Button
+            type={"button"}
+            size={"small"}
+            variation={"teritiary"}
+            label={t("ADD_OPTIONS")}
+            onClick={() =>
+              setDrawerState((prev) => ({
+                ...prev,
+                [cField?.bindTo]: prev?.[cField?.bindTo]
+                  ? [
+                      ...prev?.[cField?.bindTo],
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)